src/utils/TranslationUtils.js

```diff
@@ -37,6 +41,12 @@ const addTranslationParametersLabels = (parameters) => {
       const label = parameter.labels[lang];
       resources[lang][key] = label;
     }
+    for (const lang in parameter?.options?.tooltipText) {
+      resources[lang] = resources[lang] || {};
+      const key = getParameterTooltipTranslationKey(parameter.id);
+      const label = parameter.options.tooltipText[lang];
+      resources[lang][key] = label;
```

Comment:
Maybe we can shorten this to lines in one: `resources[lang][key] = parameter.options.tooltipText[lang] `?
It was already like that in the previous loop but we could use the occasion to refactor it a bit :slightly_smiling_face:

Reply:
Indeed. I also make this change at line 41. :+1: